locales/de-DE/translation.json

```diff
@@ -1,57 +1,58 @@
 {
-  "By default, the address is generated in a random group. You can select the group you want the address to be generated in using the Advanced options.": "Standardmäßig wird die Adresse in einer zufälligen Gruppe generiert. Sie können die Gruppe, in der die Adresse generiert werden soll, in den erweiterten Optionen auswählen.",
-  "Use an existing wallet": "Vorhandene Brieftasche verwenden",
-  "Please choose whether you want to create a new wallet or import an existing one.": "Bitte wählen Sie, ob Sie eine neue Geldbörse erstellen oder eine bestehende importieren möchten.",
-  "New wallet": "Neue Brieftasche",
-  "Import wallet": "Brieftasche importieren",
+  "By default, the address is generated in a random group. You can select the group you want the address to be generated in using the Advanced options.": "Die Adresse wird automatisch in einer zufälligen Gruppe generiert.
+   In den erweiterten Optionen können Sie die Gruppe auswählen, in der die Adresse generiert werdeden soll.",
+  "Use an existing wallet": "Vorhandene Wallet verwenden",
+  "Please choose whether you want to create a new wallet or import an existing one.": "Bitte wählen Sie, ob Sie eine neue Wallet erstellen oder eine bestehende importieren möchten.",
+  "New wallet": "Neue Wallet",
+  "Import wallet": "Wallet importieren",
   "Amount must be greater than {{ minAmountInAlph }}": "Betrag muss größer sein als {{ minAmountInAlph }}",
-  "Could not connect to the {{ currentNetwork }} network.": "Konnte keine Verbindung mit dem {{ currentNetwork }} Netzwerk herstellen.",
-  "Note that if activated, \"{{ address }}\" will not be the default address anymore.": "Beachten Sie, dass, wenn aktiviert, \"{{ address }}\" nicht mehr die Standardadresse sein wird.",
+  "Could not connect to the {{ currentNetwork }} network.": "Konnte keine Netzwerkverbindung herstellen.",
+  "Note that if activated, \"{{ address }}\" will not be the default address anymore.": "Beachten Sie, bei Aktivierung \"{{ address }}\" wird dies nicht mehr die Standardadresse.",
   "Error while sweeping address {{ from }}": "Fehler beim Durchsuchen der Adresse {{ from }}",
```

Comment:
Fehler beim Durchsuchen der Adresse.

Correction: Fehler beim entleeren der Adresse. 

"emptying/entleeren instead of "searching"/durchsuchen."